docs/reference/changelog.rst

```diff
@@ -9,6 +9,8 @@ Bug fixes:
 
 - Allow for a non-specific system Python interpreter when using the
   :ref:`uv plugin<craft_parts_uv_plugin>`.
+- Fix CPATH variable scope in the :ref:`jlink plugin<craft_parts_jlink_plugin>`.
```

Comment:
These new entries can't be under the `2.3.1` version, it's already released (February 7th) and is off of a different branch.
You need to add a new entry, like:

```
X.Y.Z (2025-MM-DD)
------------------

Bug fixes:

- Fix CPATH variable scope and Jdeps parameter ordering in the 
  :ref:`jlink plugin<craft_parts_jlink_plugin>`.

Reply:
Updated, thank you :)